CHANGELOG.md

```diff
@@ -18,6 +18,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Add new field GraphQL field `cluster{ issues {topic} }` and extend
   list of inspected topics with `replication`, `failover`, `memory`, `clock`.
+  Thresholds for issues with topic `memory` and `clock` are configurable with
+  `argparse` options. Set environment variable `TARANTOOL_DESYNC_THRESHOLD_WARNING`
+  (in seconds) to configure threshold for issues with topic `clock`. Set environment variable
+  `TARANTOOL_FRAGMENTATION_THRESHOLD_CRITICAL` and `TARANTOOL_FRAGMENTATION_THRESHOLD_WARNING`
+  (`number` in range [0, 1]) to configure thresholds for issues with topic `memory`.
```

Comment:
> (`number` in range [0, 1])

Let's add validation for this

Reply:
Also I think that TARANTOOL_FRAGMENTATION_THRESHOLD_WARNING should be less than TARANTOOL_FRAGMENTATION_THRESHOLD_CRITICAL